main.c

```diff
@@ -27,27 +38,72 @@ static void LED_init() {
     LATC7 = 1;      // turn the led off
 }
 
+void set_DACs(){
+  uint8_t data_buf[2]; 
+  data_buf[0] = ((HIGH_DAC_VAL & 0xf0) >> 4); // shift the 4 MSB to the 4 LSB
+  data_buf[1] = ((HIGH_DAC_VAL & 0x0f) << 4); // mask the 4MSB then shift the 4 LSB to the 4MSB
+  i2c1_writeNBytes(HIGH_DAC_ADDRESS, data_buf, 2);
+  data_buf[0] = ((LOW_DAC_VAL & 0xf0) >> 4); // shift the 4 MSB to the 4 LSB
+  data_buf[1] = ((LOW_DAC_VAL) << 4); // mask the 4MSB then shift the 4 LSB to the 4MSB
+  i2c1_writeNBytes(LOW_DAC_ADDRESS, data_buf, 2); 
+  if(i2c1_getLastError() == I2C1_FAIL_TIMEOUT){
+        RED_LED_ON;
+  }
+}
+
+int check_battery_voltage(void){    //returns mV
```

Comment:
Comment for where the magic number is coming from would be good (for future readers of this code).